### ISSUE TITLE
Its not uploading anything

### ISSUE THREAD

**Issue by bzemms** [gilmarsquinelato/serverless-spa#7]
this is the log tail
```
Serverless: Bucket my-app already exists
Serverless: Listing objects in bucket my-app...
Serverless: Deleting all objects from bucket my-app...
Serverless: Configuring website bucket my-app...
Serverless: Configuring policy for bucket my-app...
```

it's not enter the async waterfall readdir

**Comment by gilmarsquinelato** [comment]
Hello @bzemms, that's weird, can you check this in another machine?

**Comment by gilmarsquinelato** [comment]
This problem was solved with #8.
Fix released in v1.2.5
